src/router/index.js

```diff
@@ -0,0 +1,41 @@
+import { routes } from './appRoutes.js';
+import { httpCodes } from '../utils/httpCodes.js';
+
+export const router = (req, res) => {
+  const params = {};
+
+  function glueUrl(url, rout) {
```

Comment:
use similar style everywhere. If you use `function`, use functions everywhere.
But my personal recommend is arrow function (() =>)